### PR TITLE
Remove unimplemented SVG color-profile and solidcolor

### PR DESCRIPTION
**Pull request by ddbeck**
> What was wrong/why is this fix needed? (quick summary only)

Remove some pages and cross references to correspond to feature removals in BCD. See https://github.com/mdn/browser-compat-data/pull/10515.

> MDN URL of the main page changed

Chiefly but not exclusively:

https://developer.mozilla.org/docs/Web/API/SVGColorProfileElement
https://developer.mozilla.org/docs/Web/API/SVGSolidcolorElement
https://developer.mozilla.org/docs/Web/SVG/Element/color-profile
https://developer.mozilla.org/docs/Web/SVG/Element/solidcolor

> Issue number (if there is an associated issue)

https://github.com/mdn/browser-compat-data/pull/10515

> Anything else that could help us review it

cc: @sideshowbarker

### CODE REVIEW

File: files/en-us/web/svg/attribute/xlink_colon_arcrole/index.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
...
+<p>Twenty two elements are using this attribute: {{SVGElement("a")}}, {{SVGElement("altGlyph")}}, {{SVGElement("animate")}}, {{SVGElement("animateColor")}}, {{SVGElement("animateMotion")}}, {{SVGElement("animateTransform")}}, {{SVGElement("cursor")}}, {{SVGElement("feImage")}}, {{SVGElement("filter")}}, {{SVGElement("font-face-uri")}}, {{SVGElement("glyphRef")}}, {{SVGElement("image")}}, {{SVGElement("linearGradient")}}, {{SVGElement("mpath")}}, {{SVGElement("pattern")}}, {{SVGElement("radialGradient")}}, {{SVGElement("script")}}, {{SVGElement("set")}}, {{SVGElement("textPath")}}, {{SVGElement("tref")}}, {{SVGElement("use")}}</p>
```
Comment (chrisdavidmills) [inline files/en-us/web/svg/attribute/xlink_colon_arcrole/index.html:15]: Should be "Twenty-one elements..." if you have removed one? Also, could do with "and, " before the last item?

File: files/en-us/web/svg/attribute/xlink_colon_href/index.html [hunk 2]
```diff
@@ -14,7 +14,7 @@
...
+<p>22 elements are using this attribute: {{SVGElement("a")}}, {{SVGElement("altGlyph")}}, {{SVGElement("animate")}}, {{SVGElement("animateColor")}}, {{SVGElement("animateMotion")}}, {{SVGElement("animateTransform")}}, {{SVGElement("cursor")}}, {{SVGElement("feImage")}}, {{SVGElement("filter")}}, {{SVGElement("font-face-uri")}}, {{SVGElement("glyphRef")}}, {{SVGElement("image")}}, {{SVGElement("linearGradient")}}, {{SVGElement("mpath")}}, {{SVGElement("pattern")}}, {{SVGElement("radialGradient")}}, {{SVGElement("script")}}, {{SVGElement("set")}}, {{SVGElement("textPath")}}, {{SVGElement("tref")}}, and {{SVGElement("use")}}}</p>
```
Comment (chrisdavidmills) [inline files/en-us/web/svg/attribute/xlink_colon_href/index.html:17]: "Twenty-one"? And use words for consistency with other page?

File: files/en-us/web/svg/attribute/xlink_colon_title/index.html [hunk 3]
```diff
@@ -16,7 +16,7 @@
...
+<p>These elements are using this attribute: {{SVGElement("a")}}, {{SVGElement("altGlyph")}}, {{SVGElement("animate")}}, {{SVGElement("animateColor")}}, {{SVGElement("animateMotion")}}, {{SVGElement("animateTransform")}}, {{SVGElement("cursor")}}, {{SVGElement("feImage")}}, {{SVGElement("filter")}}, {{SVGElement("font-face-uri")}}, {{SVGElement("glyphRef")}}, {{SVGElement("image")}}, {{SVGElement("linearGradient")}}, {{SVGElement("mpath")}}, {{SVGElement("pattern")}}, {{SVGElement("radialGradient")}}, {{SVGElement("script")}}, {{SVGElement("set")}}, {{SVGElement("textPath")}}, {{SVGElement("tref")}}, and {{SVGElement("use")}}</p>
```
Comment (chrisdavidmills) [inline files/en-us/web/svg/attribute/xlink_colon_title/index.html:19]: "Twenty-one"

File: files/en-us/web/svg/attribute/xlink_colon_type/index.html [hunk 4]
```diff
@@ -10,7 +10,7 @@
...
+<p>Twenty two elements are using this attribute: {{SVGElement("a")}}, {{SVGElement("altGlyph")}}, {{SVGElement("animate")}}, {{SVGElement("animateColor")}}, {{SVGElement("animateMotion")}}, {{SVGElement("animateTransform")}}, {{SVGElement("cursor")}}, {{SVGElement("feImage")}}, {{SVGElement("filter")}}, {{SVGElement("font-face-uri")}}, {{SVGElement("glyphRef")}}, {{SVGElement("image")}}, {{SVGElement("linearGradient")}}, {{SVGElement("mpath")}}, {{SVGElement("pattern")}}, {{SVGElement("radialGradient")}}, {{SVGElement("script")}}, {{SVGElement("set")}}, {{SVGElement("textPath")}}, {{SVGElement("tref")}}, and {{SVGElement("use")}}</p>
```
Comment (chrisdavidmills) [inline files/en-us/web/svg/attribute/xlink_colon_type/index.html:13]: "Twenty-one"

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
Nice work @ddbeck . A few bits I think need looking at, but nothing much.

Also, does @sideshowbarker want to look at this one?

File: files/en-us/web/svg/attribute/xlink_colon_type/index.html (hunk 4 above)
Comment (chrisdavidmills) [reply files/en-us/web/svg/attribute/xlink_colon_type/index.html:13]: This one got missed; update it to be consistent with the others, and I think we've good to go!

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
Just one comment. I think your approach here is fine, as long as all instances are consistent. I don't think the number is important or needed.

**Review by chrisdavidmills — APPROVED** [review]
Perfect; nice one @ddbeck

**Comment by sideshowbarker** [comment]
> Also, does @sideshowbarker want to look at this one?

I post-commit reviewed it, and it all looks great to me

**Pull request merged**